d2cli/main.go

```diff
@@ -422,7 +422,7 @@ func _render(ctx context.Context, ms *xmain.State, plugin d2plugin.Plugin, sketc
 		ThemeID:       themeID,
 		DarkThemeID:   darkThemeID,
 		SetDimensions: toPNG,
-		FitScreen:     fitScreen,
+		NoFit:         noFit,
```

Comment:
Since NoFit does the same thing as `SetDimensions`, I think it is better to just have `SetDimensions: toPNG || noFit,` without another RenderOpt